File: pertemuan2/Prak/PRAK02_2473021_PY/B_TrueFalse.py
"""
Pertemuan 02
[PRAKTIKUM] Membuat Program True False

@author 2473021-Febrianus Leona Putra
@version 24 September 2024

"""
# Input
pertama = float(input("Masukkan Angka Pertama : "))
kedua = int(input("Masukkan Angka Kedua : "))
ketiga = int(input("Masukkan Angka Ketiga : "))

# Process
kondisi1 = pertama > kedua and pertama > ketiga
kondisi2 = pertama == ketiga
kondisi3 = ketiga > pertama and ketiga > kedua
kondisi4 = ketiga < kedua
kondisi5 = pertama != kedua
kondisi6 = pertama >= ketiga
kondisi7 = pertama > ketiga

# Output
print("-------------------------------------")
print("==========Program True False=========")
print("-------------------------------------")
print("Angka pertama > dari kedua dan Ketiga")
print(kondisi1)
print("-------------------------------------")
print("Angka Pertama = Angka ketiga")
print(kondisi2)
print("-------------------------------------")
print("Angka Ketiga > dari Pertama dan Kedua")
print(kondisi3)
print("-------------------------------------")
print("Angka Ketiga < dari Angka Kedua")
print(kondisi4)
print("-------------------------------------")
print("Angka Pertama != angka kedua")
print(kondisi5)
print("-------------------------------------")
print("Angka Pertama >= angka ketiga")
print(kondisi6)
print("-------------------------------------")
print("Angka Pertama > angka ketiga")
print(kondisi7)
print("-------------------------------------")

File: pertemuan2/Prak/PRAK02_2473021_JAVA/B_TrueFalse.java
package pertemuan2.Prak.PRAK02_2473021_JAVA;

import java.util.Scanner;

/**
 * Pertemuan 02
 * [PRAKTIKUM] Membuat Program True False
 * 
 * @author 2473021-Febrianus Leona Putra
 * @version 24 September 2024
 */

public class B_TrueFalse {
    public static void main(String[] args) {
        // input
        Scanner sc= new Scanner(System.in);
        System.out.print("Masukkan Angka Pertama : ");
        int pertama = sc.nextInt();
        System.out.print("Masukkan Angka Kedua : ");
        int kedua = sc.nextInt();
        System.out.print("Masukkan Angka Ketiga : ");
        int ketiga = sc.nextInt();

        // Process
        boolean kondisi1 = pertama > kedua && pertama > ketiga;
        boolean kondisi2 = pertama == ketiga;
        boolean kondisi3 = ketiga > pertama && ketiga > kedua;
        boolean kondisi4 = ketiga < kedua;
        boolean kondisi5 = pertama != kedua;
        boolean kondisi6 = pertama >= ketiga;
        boolean kondisi7 = pertama > ketiga;

        // Output
        System.out.println("-------------------------------------");
        System.out.println("==========Program True False=========");
        System.out.println("-------------------------------------");
        System.out.println("Angka pertama > dari kedua dan Ketiga");
        System.out.println(kondisi1);
        System.out.println("-------------------------------------");
        System.out.println("Angka Pertama = Angka ketiga");
        System.out.println(kondisi2);
        System.out.println("-------------------------------------");
        System.out.println("Angka Ketiga > dari Pertama dan Kedua");
        System.out.println(kondisi3);
        System.out.println("-------------------------------------");
        System.out.println("Angka Ketiga < dari Angka Kedua");
        System.out.println(kondisi4);
        System.out.println("-------------------------------------");
        System.out.println("Angka Pertama != angka kedua");
        System.out.println(kondisi5);
        System.out.println("-------------------------------------");
        System.out.println("Angka Pertama >= angka ketiga");
        System.out.println(kondisi6);
        System.out.println("-------------------------------------");
        System.out.println("Angka Pertama > angka ketiga");
        System.out.println(kondisi7);
        System.out.println("-------------------------------------");

        sc.close();
    }
}
